roles/setup/tasks/create_policy.yml

```diff
@@ -1,68 +1,68 @@
 - ansible.builtin.debug:
-    msg: 'Creating Policy {{ policy.name }}'
-  name: 'Policy Name'
+    msg: "Creating Policy {{ policy.name }}"
+  name: "Policy Name"
 - ansible.builtin.debug:
-    msg: 'Role ID {{ role_id }}'
-  name: 'Role ID'
-- name: 'Set Policy Object'
+    msg: "Role ID {{ role_id }}"
+  name: "Role ID"
+- name: "Set Policy Object"
   set_fact:
     policy_item:
-      decisionStrategy: '{{ policy.decisionStrategy }}'
-      description: '{{ policy.description }}'
-      logic: '{{ policy.logic }}'
-      name: '{{ policy.name }}'
+      decisionStrategy: "{{ policy.decisionStrategy }}"
+      description: "{{ policy.description }}"
+      logic: "{{ policy.logic }}"
+      name: "{{ policy.name }}"
       roles:
-        - id: '{{ role_id }}'
+        - id: "{{ role_id }}"
           required: true
-      type: '{{ policy.type }}'
+      type: "{{ policy.type }}"
 - debug:
-    msg: 'Policy Object {{ policy_item }}'
-  name: 'Log the policy object'
-- name: 'Add Policy'
+    msg: "Policy Object {{ policy_item }}"
+  name: "Log the policy object"
+- name: "Add Policy"
   register: create_policy
   uri:
-    body: '{{ policy_item | to_json }}'
+    validate_certs: "{{ verify_ssl }}"
+    body: "{{ policy_item | to_json }}"
     body_format: json
     headers:
       Accept: application/json
-      Authorization: 'Bearer {{ keycloak_token.json.access_token }}'
+      Authorization: "Bearer {{ keycloak_token.json.access_token }}"
       Content-type: application/json
     method: POST
     status_code:
       - 201
       - 409
-    url: '{{ auth_keycloak_url }}/admin/realms/{{ realm_name }}/clients/{{ catalog_client[''id'']
-      }}/authz/resource-server/policy/role'
-- name: 'Set Policy ID if we created a new policy'
+    url: "{{ auth_keycloak_url }}/admin/realms/{{ realm_name }}/clients/{{ catalog_client['id'] }}/authz/resource-server/policy/role"
+- name: "Set Policy ID if we created a new policy"
   set_fact:
-    policy_id: '{{ create_policy.json.id }}'
+    policy_id: "{{ create_policy.json.id }}"
   when:
-    - 'create_policy.status == 201'
-    - 'create_policy.json is defined'
-    - 'create_policy.json.id is defined'
-- name: 'Get policy if we didn''t create a new policy'
+    - "create_policy.status == 201"
+    - "create_policy.json is defined"
+    - "create_policy.json.id is defined"
+- name: "Get policy if we didn't create a new policy"
   register: get_policy_list
   uri:
+    validate_certs: "{{ verify_ssl }}"
```

Comment:
@Alex-Izquierdo Should the verify_ssl be called verify_keycloak_ssl?

Reply:
changed